js/swmp.js

```diff
@@ -310,4 +310,10 @@ disktable(jsondiskdata);
     $( function(){
       $( document ).trigger( "enhance.tablesaw" );
     });
-  })( jQuery );
\ No newline at end of file
+  })( jQuery );
+
+if(reload !== 0) {
```

Comment:
And now it doesn't matter that I hadn't added a check in the meta tag, because the meta tag is gone, and I'm doing the reload here as well.